app/grandchallenge/workstations/templatetags/workstations.py

```diff
@@ -64,4 +65,7 @@ def workstation_query(
             }
         )
 
+    if user:
+        query.update({settings.WORKSTATIONS_USER_QUERY_PARAM: user.username})
```

Comment:
This should only be passed in combination with reader study, so I think the place for it is between R40 and R41.